docs/paper/paper.md

```diff
@@ -85,12 +85,21 @@ Quantum development software packages such as Qiskit, ReCirq [@QuantumAI], tKet
 Forest [@Smith2016] are part of the quantum stack to execute quantum circuits on hardware.
 They also enable high-level applications that abstract away the quantum hardware. 
 Forest-benchmarking [@Forest] and pyGSTi [@pyGSTi] are tailored towards benchmarking of quantum hardware.
+Commercial solutions that provide quantum optimal control as a service now also exist [@Ball2021].
 However, there is still a need for open-source software that enables researchers and hardware 
 maintainers to easily execute characterization and calibration experiments.
-`Qiskit Experiments` is unique in this perspective as it provides low-level characterization 
-experiments that integrate with pulse-level control [@Alexander2020].
+Recently, software packages have started to emerge to fill this gap [@Pasquale2023]. 
+`Qiskit Experiments` is unique in this perspective as it provides open-source low-level 
+characterization experiments that integrate with pulse-level control [@Alexander2020].
```

Comment:
Is there more we can say here about what Qiskit Experiments can do that competing packages don't provide?

Reply:
I streamlined the text here: [984d8ab](https://github.com/Qiskit/qiskit-experiments/pull/1131/commits/984d8ab7dab62dd0e0860edbcc025cf80fa33d62)